xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -52,10 +53,10 @@
         <Parameter Name="value" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">String to encode.</param>
+        <summary>Encodes the supplied string and returns the encoded text as a new string.</summary>
+        <returns>>Encoded string.</returns>
```

Comment:
There's an extra `>` before the sentence that is making this sentence render as a note.